### PR TITLE
init script gentoo

### PR DESCRIPTION
**Pull request by lunny**
cherry-pick from gogits/gogs#3761

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by tboerger** [comment]
Fixes https://github.com/go-gitea/gitea/issues/432

File: scripts/init/gentoo/gitea [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/sbin/openrc-run
+
+DIR=/home/git/gitea
+USER=git
+PORT=3000
```
Comment (Bwko) [inline scripts/init/gentoo/gitea]: Why override the app.ini port number?

Comment (lunny) [reply scripts/init/gentoo/gitea]: Just cherry-pick, don't know.

Comment (Bwko) [reply scripts/init/gentoo/gitea]: Could you remove it?

Comment (lunny) [reply scripts/init/gentoo/gitea]: done.

File: scripts/init/gentoo/gitea [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/sbin/openrc-run
+
+DIR=/home/git/gitea
+USER=git
+
+start_stop_daemon_args="--user ${USER} --chdir ${DIR}"
+command="${DIR}/gitea"
+command_args="web -port ${PORT}"
```
Comment (tboerger) [inline scripts/init/gentoo/gitea]: `command_args="web"` :)

**Comment by tboerger** [comment]
Still LGTM

**Comment by Bwko** [comment]
LGTM

**Pull request merged**